ibc-testkit/src/utils/mod.rs

```diff
@@ -0,0 +1,10 @@
+use ibc::primitives::Timestamp;
+use tendermint::Time;
+
+/// Returns a `Timestamp` representation of beginning of year 2023.
```

Comment:
Would it be beneficial to include a brief explanation, perhaps one or two sentences, on why this is defined and utilized in tests.

Reply:
1561834f